### ISSUE TITLE
release package does not work from outside of the installed directory

### ISSUE THREAD

**Issue by suyograo** [elasticsearch/logstash#2714]
Installed the deb package which was created from another machine on to a clean machine. 

```
sudo dpkg -i logstash_1.5.0.beta2-1_all.deb
ubuntu@ip-10-0-1-9:~$ sudo /opt/logstash/bin/logstash -e 'input { generator {} } '
Bundler::GemfileNotFound: Could not locate Gemfile or .bundle/ directory
                         root at /opt/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.8.3/lib/bundler.rb:198
                  bundle_path at /opt/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.8.3/lib/bundler.rb:101
  configure_gem_home_and_path at /opt/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.8.3/lib/bundler.rb:420
                    configure at /opt/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.8.3/lib/bundler.rb:92
                   definition at /opt/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.8.3/lib/bundler.rb:153
                        setup at /opt/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.8.3/lib/bundler.rb:118
               bundler_setup! at /opt/logstash/lib/logstash/environment.rb:104
                       (root) at /opt/logstash/lib/logstash/runner.rb:8
```

but with going inside the directory and running it from there, it works

```
cd /opt/logstash
sudo bin/logstash -e 'input { generator {} } '
```

**Comment by ph** [comment]
Content of the `.bundle/config`
```
ubuntu@ip-10-0-1-9:/opt/logstash$ cat /opt/logstash/.bundle/config
---
BUNDLE_PATH: /opt/logstash/vendor/bundle
BUNDLE_GEMFILE: /opt/logstash/Gemfile
BUNDLE_WITHOUT: development
```

**Comment by jordansissel** [comment]
Same for rpm (as expected)

```
[root@centos logstash]# less /var/log/logstash/logstash.err
Bundler::GemfileNotFound: Could not locate Gemfile or .bundle/ directory
                         root at /opt/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.8.3/lib/bundler.rb:198
                  bundle_path at /opt/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.8.3/lib/bundler.rb:101
  configure_gem_home_and_path at /opt/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.8.3/lib/bundler.rb:420
                    configure at /opt/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.8.3/lib/bundler.rb:92
                   definition at /opt/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.8.3/lib/bundler.rb:153
                        setup at /opt/logstash/vendor/bundle/jruby/1.9/gems/bundler-1.8.3/lib/bundler.rb:118
               bundler_setup! at /opt/logstash/lib/logstash/environment.rb:104
                       (root) at /opt/logstash/lib/logstash/runner.rb:8
```

**Comment by colinsurprenant** [comment]
solved in #2717

**Comment by suyograo** [comment]
Fixed
